api/v1alpha1/watermarkpodautoscaler_types.go

```diff
@@ -66,6 +66,10 @@ type WatermarkPodAutoscalerSpec struct {
 	// ScaleDownLimitFactor == 0 means that downscaling will not be allowed for the target.
 	ScaleDownLimitFactor *resource.Quantity `json:"scaleDownLimitFactor,omitempty"`
 
+	// Number of replicas to scale by at a time. When set, replicas added or removed must be a multiple of this parameter.
+	// Allows for special scaling patterns, for instance when an application requires a certain number of pods in multiple
+	ReplicaScalingInterval *int32 `json:"replicaScalingInterval,omitempty"`
```

Comment:
Could you add the followig `kubebuilder` comment.

```suggestion
         // +kubebuilder:validation:Minimum=1
	ReplicaScalingInterval *int32 `json:"replicaScalingInterval,omitempty"`
```

Also `ScaleReplicaFactor` can be a better name. 
@CharlyF WDYT?